UnitTests/Tests/Controllers/TRApplicationControllerSpec.m

```diff
@@ -0,0 +1,61 @@
+@import CoreLocation;
+#import "TRApplicationController.h"
+#import "TRWeatherController.h"
+#import "TRLocationController.h"
+
+@interface TRApplicationController (Tests)
+
+@property (nonatomic) TRWeatherController *weatherController;
+@property (nonatomic) TRLocationController *locationController;
+
+@end
+
+@interface TRWeatherController (Tests)
+
+@property (nonatomic) RACCommand *updateWeatherCommand;
+
+@end
+
+SpecBegin(TRApplicationController)
+
+describe(@"TRApplicationController", ^{
+    describe(@"setMinimimBackgroundFetchIntervalForApplication:", ^{
+        it(@"sets the interval to minimum when authorization is always", ^{
+            UIApplication *app = OCMClassMock([UIApplication class]);
+            TRLocationController *locationController = OCMPartialMock([[TRLocationController alloc] init]);
+            OCMStub([locationController authorizationStatusEqualTo:kCLAuthorizationStatusAuthorizedAlways]).andReturn(YES);
+            TRApplicationController *ctrl = [TRApplicationController new];
+            ctrl.locationController = locationController;
+
+            [ctrl setMinimumBackgroundFetchIntervalForApplication:app];
+
+            OCMVerify([app setMinimumBackgroundFetchInterval:UIApplicationBackgroundFetchIntervalMinimum]);
+        });
+
+        it(@"sets the interval to never when authorization is not always", ^{
+            UIApplication *app = OCMClassMock([UIApplication class]);
+            TRLocationController *locationController = OCMPartialMock([[TRLocationController alloc] init]);
+            OCMStub([locationController authorizationStatusEqualTo:kCLAuthorizationStatusAuthorizedAlways]).andReturn(NO);
+            TRApplicationController *applicationController = [TRApplicationController new];
```

Comment:
In the the spec above you call this same thing `ctrl`. What do you think about making the variables in the setups, since they are identical, named the same thing?